### PR TITLE
bugfix:copyright check

### LINKED ISSUE #887: bugfix:copyright check
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description

copyright check `Copyright 1999-2019 Seata.io Group`

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
Just paste your stack trace here!
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

### Ⅵ. Environment:

- JDK version :
- OS :
- Others:

### PR DESCRIPTION
**Pull request by slievrly**
### Ⅰ. Describe what this PR did
bugfix:copyright check

### Ⅱ. Does this pull request fix one issue?
fix #887

### CODE REVIEW

**Comment by leizhiyuan** [comment]
for properties  file
maybe it should be 

```
        <property name="header" value="^.*$\n#[\s]*Copyright 1999-2019 Seata.io Group."/>
```

image

File: style/feats_checkstyle.xml [hunk 1]
```diff
@@ -28,12 +28,12 @@
...
     <!-- header -->
     <module name="RegexpHeader">
-        <property name="header" value="^.*$\n[\s]Copyright 1999-2018 Alibaba Group Holding Ltd."/>
+        <property name="header" value="^.*$\n[\s]Copyright 1999-2019 Seata.io Group."/>
         <property name="fileExtensions" value="java"/>
     </module>
 
     <module name="RegexpHeader">
-        <property name="header" value="#[\s]*Copyright 1999-2018 Alibaba Group Holding Ltd.*"/>
+        <property name="header" value="^.*$\n[\s]Copyright 1999-2019 Seata.io Group."/>
```
Comment (leizhiyuan) [inline style/feats_checkstyle.xml]: for properties , it should start # xxx

Comment (slievrly) [reply style/feats_checkstyle.xml]: > for properties , it should start # xxx

thanks, i will check.

**Comment by leizhiyuan** [comment]
LGTM

**Pull request merged**